pages/treeherder.py

```diff
@@ -93,6 +93,9 @@ class ResultSet(PageRegion):
         _datestamp_locator = (By.CSS_SELECTOR, '.result-set-title-left > span a')
         _jobs_locator = (By.CLASS_NAME, 'job-btn')
         _pin_all_jobs_locator = (By.CLASS_NAME, 'pin-all-jobs-btn')
+        _dropdown_toggle_locator = (By.CSS_SELECTOR, 'button.btn.btn-sm.btn-resultset.dropdown-toggle')
```

Comment:
This locator is improved, but you should use the locator here: 
https://github.com/mozilla/treeherder-tests/blob/master/pages/treeherder.py#L95